### DISCUSSION TITLE
Grafana Spec file

### DISCUSSION THREAD

**Discussion by Kamlesh356** [grafana/grafana#55987] [General]
Can someone here help to get the grafana.spec file for version 9.1.6

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
